src/ODC-By-1.0.xml

```diff
@@ -0,0 +1,504 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="ODC-By-1.0"
+            name="Open Data Commons Attribution License v1.0">
+      <crossRefs>
+         <crossRef>https://opendatacommons.org/licenses/by/1.0/</crossRef>
+      </crossRefs>
+    <text>
+      <titleText>
+	     <optional>##</optional>
```

Comment:
I think we should use `<alt>` for these.  For an example of a similar change, see #661.

Reply:
discussed on call as could go either way - this was what Gary submitted and it's works, so leaving as is (and copied same method here) unless it's so annoying or causes problems that someone wants to update :)

Reply:
> ... unless it's so annoying or causes problems that someone wants to update :)

I'm fine filing a follow-up update like #661.